app/views/frontend/conferences/show.html.haml

```diff
@@ -13,11 +13,11 @@
       %h2= @conference.acronym.gsub(/[\-_]/,' ')
 
     #sorting.btn-group.btn-group-sm.btn-group-justified
-      - %w{name duration date}.each do |sorting|
+      - %w{name duration date view_count}.each do |sorting|
         - if @sorting == sorting
-          %a.btn.btn-primary.active{ href: conference_path(acronym: @conference.acronym, sort: sorting) }= sorting
+          %a.btn.btn-primary.active{ href: conference_path(acronym: @conference.acronym, sort: sorting) }= sorting.gsub(/_/, ' ')
```

Comment:
Better use a lookup here for Internal -> Display mapping. Would be more natural if only we used translation files…